.ci/pull_syc_fluent_images.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Cleaning Cached Images Off Runner"
```

Comment:
```suggestion
echo "Cleaning cached images off runner."
```
Messages should be written with a subject, verb, and concluding punctuation. Should "off" be "of" instead? It seems so.